Userland/Libraries/LibWeb/HTML/Scripting/TemporaryExecutionContext.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Forward.h>
+#include <LibWeb/Forward.h>
+
+namespace Web::HTML {
+
+// FIXME: When JS is run from outside the context of any user script, we currently do not have a running execution
```

Comment:
I don't think this FIXME is actionable, these are spec bugs after all.

Reply:
Sure. I just made it a FIXME since the existing workarounds are also FIXMEs.